### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: BUGFIX
+assignees: ''
+
+---
+
+## 🐞 bug report
```
Comment (markwhitfeld) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
## 🐞 Bug Report
```

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: FEATURE
+assignees: ''
+
+---
+
+## 🚀 feature request
```
Comment (markwhitfeld) [inline .github/ISSUE_TEMPLATE/feature-request.md]:
```suggestion
## 🚀 Feature Request
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+## 🔬 Minimal Reproduction
+<!-- Please create and share minimal reproduction -->
+<!-- ✍️--> https://stackblitz.com/...
+
+<!--
+If StackBlitz is not suitable for reproduction of your issue, please create a minimal GitHub repository with the reproduction of the issue.
+A good way to make a minimal reproduction is to create a new app via `ng new repro-app` and add the minimum possible code to show the problem. 
+Share the link to the repo below along with step-by-step instructions to reproduce the problem, as well as expected and actual behavior.
+-->
+
```
Comment (markwhitfeld) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Please add the bit I added today right here. This should not be a comment but actual text that the user must delete once read.
```suggestion
**Note:** Our policy is that issues reported without a reproduction will be closed immediately and then reopened once a reproduction has been provided. Please respect the developers of this project by doing this. We give of our personal time to work on this project and would rather be spending our time fixing or enhancing the library than chasing down badly described or unreproducible issues.
Please delete this note once you have read it.

```

**Comment by markwhitfeld** [comment]
Ah, looking at the angular Repo, I get how this will work.
Did you take these snippets from there?
Could we add templates for each of these types:
```
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => https://github.com/ngxs/store/blob/master/CONTRIBUTING.md
[ ] Other... Please describe:
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+## 🌍  Your Environment
+
+**Angular Version:**
```
Comment (markwhitfeld) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Where do we ask for the version of NGXS?

Comment (markwhitfeld) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Browser versions too?

**Comment by splincode** [comment]
@markwhitfeld > Could we add templates for each of these types

Why, we divided it into several files, why do we need this template?

**Comment by markwhitfeld** [comment]
So, of these issue types that we had in the original template we have only covered 2 of these scenarios:
```
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Performance issue
[X] Feature request
[ ] Documentation issue or request
[ ] Support request => https://github.com/ngxs/store/blob/master/CONTRIBUTING.md
[ ] Other... Please describe:
```
I don't mean that we would add this at the top of the template, but that we have a template for each of the other types here (similar to the angular repo), otherwise, what would someone use to log a regression, docs, performance, support request or other issue?

**Pull request closed without merging**